onyo/cli/get.py

```diff
@@ -55,13 +55,21 @@
         default=None,
         help=r"""
             Criteria to match in the form ``KEY=VALUE`` — where **VALUE** is a
-            literal string or a python regular expression. All keys can be
+            literal string or a python regular expression. Valid operands are
```

Comment:
```suggestion
            literal string or a python regular expression. Valid operations are
```

Or just "comparisons"?

Reply:
I chose operator. Fixed in the latest push.